conda_rpms/install.py

```diff
@@ -301,6 +301,10 @@ def create_meta(prefix, dist, info_dir, extra_info):
     meta_dir = join(prefix, 'conda-meta')
     if not isdir(meta_dir):
         os.makedirs(meta_dir)
+        # Lock down the conda-meta directory, which may contain
+        # API credentials.
+        mode = S_IRUSR | S_IWUSR | S_IXUSR
+        os.chmod(meta_dir, mode)
```

Comment:
It would help to add a comment to say this is not a part of the original code that we got from conda, see: https://github.com/SciTools-incubator/conda-rpms/blob/56a232c2dea2000604dc73367aa4e965ba03c946/conda_rpms/install.py#L8-L11

Reply:
@lbdreyer Done. Updated the header with appropriate details, and added markers in the `create_meta` function.